servers/rendering/renderer_rd/effects_rd.cpp

```diff
@@ -1365,15 +1370,19 @@ void EffectsRD::cubemap_filter(RID p_source_cubemap, Vector<RID> p_dest_cubemap,
 	RD::get_singleton()->compute_list_end();
 }
 
-void EffectsRD::render_sky(RD::DrawListID p_list, float p_time, RID p_fb, RID p_samplers, RID p_fog, PipelineCacheRD *p_pipeline, RID p_uniform_set, RID p_texture_set, const CameraMatrix &p_camera, const Basis &p_orientation, float p_multiplier, const Vector3 &p_position) {
+void EffectsRD::render_sky(RD::DrawListID p_list, float p_time, RID p_fb, RID p_samplers, RID p_fog, PipelineCacheRD *p_pipeline, RID p_uniform_set, RID p_texture_set, const CameraMatrix &p_left_eye_projection, const CameraMatrix &p_right_eye_projection, const Basis &p_orientation, float p_multiplier, const Vector3 &p_position) {
```

Comment:
For sky rendering, I think using left and right makes not much sense since for the most part you already made the most code N-views capable. I would fix this for consistency.